src/permission.svc.js

```diff
@@ -121,6 +122,13 @@
 
             return definedPermissions;
           },
+          defineDefaultPermissions: function (permissions) {
+            defaultPermissions = permissions;
+            return Permission;
+          },
+          defaultPermissions: function () {
```

Comment:
I'd change this, as to me it doesn't make much sense, specially since you are accessing the value from within the service. Either way is correct, but lets change it for consistency